src/main/java/HW_5/myCity/Building.java

```diff
@@ -0,0 +1,7 @@
+package HW_5.myCity;
+
+import HW_5.myClass.Person;
+
+public class Building {
+    public Person[] persones;
```

Comment:
1) mistake in "persones" word
2) Please add some information to Person class(name, surname, age, toString()), create few persons in main method in Building class, add these persons to array and using cycle display persons from array

Reply:
fixed